packages/zod-firebase/src/collection/factory/single-document-collection-factory.ts

```
@@ -13,7 +13,11 @@ import {
   type SchemaWriteDocumentReference,
 } from '../../schema'
 
-import { multiDocumentCollectionFactory, type SchemaFallbackValue } from './multi-document-collection-factory'
+import {
+  multiDocumentCollectionFactory,
+  type SchemaFallbackOutputDocument,
+  type SchemaFallbackValue,
+} from './multi-document-collection-factory'
```

ESLint

[error] Unable to resolve path to module './multi-document-collection-factory'.

(import-x/no-unresolved)